classes/Sensors/System.php

```diff
@@ -210,7 +210,7 @@ public function EventAdminInit()
         if ($action == 'update' && isset($_REQUEST['_bbp_default_role'])) {
             $oldRole = get_option('_bbp_default_role');
             $newRole = $_REQUEST['_bbp_default_role'];
-            if ($oldRole !== $newRole) {
+            if ($oldRole != $newRole) {
```

Comment:
@carmelopreeostudios are you sure about this change? like this we're not checking the data type of the variables. is that ok?